js/components/eiti-bar-chart.js

```diff
@@ -268,14 +264,6 @@
 
     xAxis.selectAll('path, line')
         .attr('fill', 'none');
-
-    svg.append('g')
-      .attr('class', 'x-axis-label')
-      .append('text')
-        .text(xAxisLabel)
-        .attr('transform', function(d) {
-          return 'translate(' + [labelOffset, fullHeight] + ')';
-        });
```

Comment:
This is where you would remove the x-axis-label, but I'm not seeing that it has been removed :/